packages/examples/cvat/recording-oracle/alembic/versions/1e89224ad721_add_gt_labels_in_gt_stats.py

```diff
@@ -0,0 +1,40 @@
+"""add_gt_labels_in_gt_stats
```

Comment:
Looks like a duplication of:

https://github.com/humanprotocol/human-protocol/blob/develop/packages/examples/cvat/recording-oracle/alembic/versions/1e89224ad721_add_gt_labels_in_gt_stats.py

Reply:
It was from the dependency PR. No more shown after merging with develop.